omnichannel-server/src/feature/test/MessageSourceController.test.ts

```diff
@@ -1,11 +1,19 @@
-import { Dependencies } from "@corecodeio/libraries/di";
-import { MessageSourceControllerInjectionKey } from "../message-source/InjectionKeys";
-
-const dependencies = new Dependencies();
-const messageSourceDigest = dependencies.provide(
-  MessageSourceControllerInjectionKey
-);
+import axios from 'axios'
 
 describe("Testing MessageSourceController messageSource method", () => {
-  test("Should return something", () => {});
+  test("Should return something", async (done) => {    
+    await axios.post('http://localhost:8001/message-source',{
+      currentTime: new Date(),
+      message: "Hello World Tests",
+      from: "+123456789",
+    }, {
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    }).then(res => {
+      expect(res.status).toBe(200);
+      done();
+    })
+    .catch(done.fail)
```

Comment:
Buen acercamiento. 

Un tip: si usas `async / await` no necesitas usar `then/catch`: 

```suggestion
     try {
    const response = await axios.post('http://localhost:8001/message-source',{
      currentTime: new Date(),
      message: "Hello World Tests",
      from: "+123456789",
    }, {
      headers: {
        'Content-Type': 'application/json'
      }
    });
    expect(response.status).toBe(200);     
     } catch (error) {
      // do something with the error. 
     }
```

En este ejemplo, no confundir `try/catch` con `then/catch`. ☝️

Reply:
a ok voy a probar hacer ese cambio